### ISSUE TITLE
Why does the new version not support mongodb

### ISSUE THREAD

**Issue by 12gameRensheng** [strapi/strapi#14610]
## Why does the new version not support mongodb
  + How can the new version support mongodb
  + thank you  
![image](https://user-images.githubusercontent.com/45571957/195504053-a149aa05-ec0e-4b45-ba26-2a9c78b5dbb9.png)

### Native environment and version
```
{
  "name": "mysql",
  "private": true,
  "version": "0.1.0",
  "description": "A Strapi application",
  "scripts": {
    "develop": "strapi develop",
    "start": "strapi start",
    "build": "strapi build",
    "strapi": "strapi"
  },
  "devDependencies": {},
  "dependencies": {
    "@strapi/strapi": "4.4.3",
    "@strapi/plugin-users-permissions": "4.4.3",
    "@strapi/plugin-i18n": "4.4.3",
    "mysql": "2.18.1"
  },
  "author": {
    "name": "A Strapi developer"
  },
  "strapi": {
    "uuid": "3fdf4f62-4ad1-4db6-822a-a45bbfd33a27"
  },
  "engines": {
    "node": ">=14.19.1 <=18.x.x",
    "npm": ">=6.0.0"
  },
  "license": "MIT"
}
```

**Comment by gu-stav** [comment]
Strapi has explained the reasons behind that change in several places: https://github.com/strapi/strapi/issues/10186

tl;dr: SQL and No-SQL databases have important differences and the way Strapi works is more aligned with the former.

If you have questions regarding the product please use Discord or the Forum. Issues are meant for bugs.
